### DISCUSSION TITLE
Tufte-Quarto project type

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#3062] [Show and Tell]
I would like to let you know that I just released [Tufte-Quarto](https://fredguth.github.io/tufte-quarto/) project type. I have mentioned it before, but now I added some documentation. 

Many thanks to the community for all the support here in this forum.

**Comment by jwang-lilly** [comment]
@fredguth, Tufte-Quarto book type is beautiful.  Could it be used for default project type, not a book?

**Reply by fredguth** [reply]
Sure. You can just use it for a website. It will work out of the box. If you have any difficulties share them and I will try to help.
